src/locales/en_US.js

```diff
@@ -590,7 +591,8 @@ const strings = {
   fio_address_register_default_fio_wallet_name: 'FIO Wallet',
   fio_address_register_no_wallet_name: 'No wallet name',
   fio_address_register_select_wallet: 'Connect to FIO Wallet',
-  fio_address_register_pending: 'Your FIO Address will be registered as soon as payment is received. Tap FIO Address in few minutes.',
+  fio_address_register_pending: 'Your FIO Address will be registered as soon as payment is received. Tap FIO Names in few minutes.',
+  fio_address_register_pending_free: 'Your free FIO Address will be registered soon. Tap FIO Names in few seconds.',
```

Comment:
Your free FIO Address will be registered in a few seconds. Tap FIO Names to see your address.